sections/builder.js

```diff
@@ -1356,6 +1356,39 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "intersect",
+    example: ".intersect([*queries], [wrap])",
```

Comment:
what does the * in *queries stand for?

Reply:
I believe that is denoting that we can call intersect like
```
knex(tableName).intersect(callback1, callback2, callback3)
```

There is similar syntax using the `*` in `select`, `union`, `dropColumns`, and `groupBy`

Reply:
Ah, I see, this means "variable amount of".